Bindings/Java/OpenSimJNI/CMakeLists.txt

```diff
@@ -114,15 +114,18 @@ set_source_files_properties(
         PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS}")
 
 if(${OPENSIM_USE_INSTALL_RPATH})
```

Comment:
Here is the one place where we set this variable: https://github.com/opensim-org/opensim-core/blob/cmake_install_dependencies_BTK_RPATH/CMakeLists.txt#L520